third_party/parser/cc/grammars/typedruby.ypp

```diff
@@ -1598,6 +1598,7 @@ int yylex(parser::semantic_type *lval, ruby_parser::typedruby27 &driver) {
                       driver.lex.cmdarg.push(false);
                       driver.lex.cond.push(false);
                       driver.lex.context.push(Context::State::DEF);
+                      driver.current_arg_stack.push("");
```

Comment:
We don't care about the name but we want to make sure we will `pop` the correct thing later.